app/views/past_foreign_secretaries/index.html.erb

```diff
@@ -172,612 +172,610 @@
 </div>
 
 <div class="govuk-grid-row featured-profiles foreign-secretaries">
-  <header class="govuk-grid-column-one-quarter govuk-!-padding-0 govuk-!-padding-top-3 heading">
+  <header class="govuk-grid-column-full govuk-!-padding-left-0 govuk-!-padding-bottom-3 heading">
     <h3>Foreign Secretaries <span>1782 to present</span></h3>
```

Comment:
Might be easier to take this heading out of the grid row and put it above it - then you don't have to fight with all the override classes.

Reply:
Done.